.travis.yml

```diff
@@ -14,20 +14,12 @@ notifications:
 jobs:
   include:
     - stage: test
-      name: "Testing Stage"
+      name: "Build and deploy"
       script: bash travis_test
-    - stage: docker
-      name: "Deploy Image Stage"
+      after_success:
```

Comment:
This currently pushes the docker image if the tests pass; ideally we'd have it push the image when we're ready to deploy instead. Switching this to a `before_deploy` stage would ensure that it's only performed if we're actually about to deploy

Reply:
updated